packages/Progress/index.tsx

```diff
@@ -0,0 +1,109 @@
+import React, {useMemo, useRef} from 'react';
+import styled from '@emotion/native';
+import Svg, {Circle} from 'react-native-svg';
+import {Animated, TextStyle, ViewStyle} from 'react-native';
+import {useTheme, DoobooTheme, withTheme} from '@dooboo-ui/theme';
+
+type DoobooThemeContext = {theme: DoobooTheme};
+type ProgressType = 'success' | 'danger' | 'warning' | 'info';
+interface Props {
+  styles?: {
+    container?: ViewStyle;
+    text?: TextStyle;
+  };
+  style?: ViewStyle;
+  size?: number;
+  radius?: number;
+  color?: string;
+  type?: ProgressType;
+  bgColor?: string;
+  progress: number;
+  strokeWidth?: number;
```

Comment:
Anything related to style should be nested in `styles` or `style`. That way we can simplify the components. You can get the idea on [dooboo-ui coding guideline](https://dooboo-ui.dooboolab.com/?path=/docs/contributing-contributing--page#coding-guidelines).

I think here we can remove `radius`, `color`, `bgColor` and `strokeWidth` and have default style instead.

Reply:
Also I think `size` can be renamed to `value`.

Reply:
I moved the props ```radius```, ```color```, ```strokeWidth``` to inside of ```styles```. And I removed ```size``` and ```bgColor``` because those are not used. Is this the right way to change it?